crates/bevy_ecs/src/system/commands/mod.rs

```diff
@@ -647,6 +703,37 @@ where
     }
 }
 
+pub struct MoveBundle<T> {
+    pub source: Entity,
+    pub target: Entity,
+    pub _bundle: PhantomData<T>,
```

Comment:
Not a big fan of `PhantomData` fields being public. Make these private and then add a `new` constructor?